compiler/expr_visitor_test.py

```diff
@@ -132,6 +140,8 @@ def foo(a, b=2):
   testDictEmpty = _MakeLiteralTest({})
   testDictNonEmpty = _MakeLiteralTest({'foo': 42, 'bar': 43})
 
+  testSetNoneEmpty = _MakeLiteralStringTest({'foo', 'bar'})
```

Comment:
I think you want to pass a string value into _MakeLiteralStringTest() right? otherwise this value will be converted to the string "set(['foo', 'bar'])" which will be passed into _GrumpRun().

Otherwise, I think this approach is good, and yes, I think all the literal tests should work this way. So maybe just change the way _MakeLiteralTest works.